reark/src/main/java/io/reark/reark/data/stores/StoreGetInterface.java

```diff
@@ -0,0 +1,10 @@
+package io.reark.reark.data.stores;
+
+import rx.Observable;
+
+/**
+ * Created by ttuo on 27/06/16.
+ */
+public interface StoreGetInterface<T, U> {
+    Observable<U> getStream(T id);
```

Comment:
I think we should have get() that returns a value if it exists, and then completes. Stream with no values that doesn't complete can't be used e.g. with .first() to get the current value